Crashlytics/generate_project.sh

```diff
@@ -19,4 +19,4 @@ readonly DIR="$( git rev-parse --show-toplevel )"
 
 "$DIR/Crashlytics/ProtoSupport/generate_crashlytics_protos.sh" || echo "Something went wrong generating protos.";
 
-pod gen "${DIR}/FirebaseCrashlytics.podspec" --auto-open --gen-directory="${DIR}/gen" --local-sources="${DIR}" --platforms=ios,macos,tvos,watchos --clean
+pod gen "${DIR}/FirebaseCrashlytics.podspec" --auto-open --gen-directory="${DIR}/gen" --local-sources="${DIR}" --platforms=ios,macos,tvos --clean
```

Comment:
Can you revert this change?

Reply:
Reverted! I need this for generate_project.sh to run locally - I haven't looked deeper into any setup I might've missed, but is that a known issue?

Reply:
Yeah I think there's some weird interaction with the podspec